_posts/2016/2016-02-09-microservice-meetup.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "epages hosts Microservices Meetup"
+date: "2016-02-09 07:16:34"
+icon: cc-discover
+categories: events
+authors: ["Mathias"]
+---
+
+Full house at epages in Hamburg!
+Last Wednesday, we hosted the [Microservices Meetup Hamburg](http://www.meetup.com/de-DE/Microservices-Meetup-Hamburg/events/224965581/?a=socialmedia).
+About 30 developers accepted the invitation and came to listen to the story about our journey through the world of Service Discovery.
+
+{% image blog-microservice-meetup-1.jpg %}{% endimage %}
+
+We had two presentation slots that filled the evening.
+And next to soft drinks, beer and pizza, there was also enough time to talk shop.
+
+With microservices we are pushing complexity from the application into the infrastructure. One of the complexities that you find yourself confronted with in the world of microservices is service discovery.
```

Comment:
Reword: Service discovery is one of the complexities that you find yourself confronted with in the world of microservices.